app/views/regions/v1/add-organisation.html

```diff
@@ -10,16 +10,14 @@
 
       {{ backLink({ href: "/regions/v1", text: "Back" }) }}
 
-      <h1 class="nhsuk-heading-xl">Invite an organisation</h1>
-
-      <p>Invite an organisation to set up an account first. Later, you can activate it to allow them to record vaccinations.</p>
+      <h1 class="nhsuk-heading-xl">Find an organisation to invite</h1>
 
       <form action="/regions/v1/organisation-details" method="post">
 
       <div class="nhsuk-form-group">
         <h1 class="nhsuk-label-wrapper">
           <label class="nhsuk-label nhsuk-label--m nhsuk-u-margin-bottom-1" for="organisationCode">
-            NHS trust or community pharmacy
+            NHS organisation
```

Comment:
I’m not sure about this, as do community pharmacies count as NHS organisations? 🤔 I thought they were independent?

Reply:
Maybe we just say Organisation - that covers NHS trusts, pharmacies, maternity settings (and others that we may add in future like GPs)

Reply:
Good shout, let's change to Organisation.

From my current understanding, maternity settings wouldn't get listed here, as they'd be part of an NHS Trust rather than a separate organisation.